config/migrations/2022/20220829135928-erediesten-data/20220928092536-clear-erediensten-governing-bodies-before-2023.sparql

```diff
@@ -0,0 +1,49 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX lblodOrg: <http://lblod.data.gift/vocabularies/organisatie/>
+PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/>
+PREFIX regorg: <http://www.w3.org/ns/regorg#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX schema: <http://schema.org/>
+PREFIX locn: <http://www.w3.org/ns/locn>
+
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?bestuursorgaanInTime ?pbestuursorgaanInTime ?obestuursorgaanInTime .
+  }
+}
+WHERE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    VALUES ?type {
+      ere:BestuurVanDeEredienst
+      ere:CentraalBestuurVanDeEredienst
+    }
+  
+    ?bestuurseenheid a ?type .
+  
+    ?bestuursorgaan besluit:bestuurt ?bestuurseenheid .
+  
+    ?bestuursorgaanInTime mandaat:isTijdspecialisatieVan ?bestuursorgaan ;
+      mandaat:bindingStart ?start ;
+      ?pbestuursorgaanInTime ?obestuursorgaanInTime .
+
+    # We just keep the ones we created for time periods 2023-2026 and 2026-2029
+    FILTER( ?start < "2023-04-01T00:00:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> )
```

Comment:
According to #253, Organen dates are not `dateTime` anymore right ?

Reply:
Indeed but this one will run before #252 (see timestamps) so when it'll run, it'll still be dateTimes and not dates yet !